tech.json

```diff
@@ -1901,7 +1901,17 @@
               ],
               "otherRequires": [],
               "note": "Using the Samus echoes that are emitted after a shinespark bonk as a weapon to kill enemies."
-            }
+            },
+            {
+              "name": "canControlShinesparkEnd",
+              "techRequires": [
+                "canShinespark"
```

Comment:
I think this should say that you end the strat with at most 29 energy

Reply:
I think the way that we express the energy requirement may be a bit situational. So I'm not sure it makes sense to include as a requirement inside the tech. It's good to keep in mind though everywhere it is used

Reply:
Oh, yeah, I was confused. It's needed in one of the strats I'm writing, but you're right, its not always or even usually needed.